examples/marko-cli/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "marko-cli",
+  "version": "1.0.0",
+  "description": "Demo of how to build an app using marko-starter",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marko-js-samples/marko-starter-demo"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@storybook/marko": "4.0.0-alpha.1",
+    "@storybook/addon-actions": "4.0.0-alpha.4",
+    "@storybook/addon-backgrounds": "4.0.0-alpha.4",
+    "@storybook/addon-centered": "4.0.0-alpha.4",
+    "@storybook/addon-knobs": "4.0.0-alpha.4",
+    "@storybook/addon-links": "4.0.0-alpha.4",
+    "@storybook/addon-notes": "4.0.0-alpha.4",
+    "@storybook/addon-options": "4.0.0-alpha.4",
```

Comment:
I see these addons here. Do all of them supported and do they all of them have examples here?

Reply:
Not yet. But plan is to add support soon!